src/gui/virtualstudio.h

```diff
@@ -297,12 +319,22 @@ class VirtualStudio : public QObject
 
     Meter* m_inputMeter;
     Meter* m_outputMeter;
+    Meter* m_inputTestMeter;
+    Volume* m_inputVolumePlugin;
+    Volume* m_outputVolumePlugin;
     QTimer m_inputClipTimer;
     QTimer m_outputClipTimer;
 
     float m_meterMax = 0.0;
     float m_meterMin = -64.0;
 
+    float m_inMultiplier  = 0.7;
```

Comment:
is there a reason this is not `1.0`?

Reply:
Not any real reason other than I was testing if it was working when I changed it. I'll change it back